### ISSUE TITLE
Capability request for the Site Notification Banner 

### ISSUE THREAD

**Issue by dghweb** [uweb/uw_wp_theme#58]
Hi,

Can access to the Site Notification Banner (under Appearance) be allowed for the [edit_theme_options](https://wordpress.org/support/article/roles-and-capabilities/#edit_theme_options) capability.

It currently does not show as a menu item for users that do not have the administrator role (or users with the "manage_options" capability) in their role but do have "edit_theme_options" in their role. I think it should be a feature that is allowed for users that have the "edit_theme_options" capability in their role.

As an administrator for dozens of websites I don't want to manage all the banner notifications on these sites and I'd welcome the possibility to delegate this down to selected site users that are set with the "edit_theme_options" capability.

Thanks for taking this in consideration.

Jasper Bleijs
Dept of Global Health

**Comment by lcaple** [comment]
Hey Jasper,

After having a discussion with the devs on the team, we've decided not to make this change. In WordPress, edit_theme_options is an administrator privilege and we don't want to break anything beyond WordPress' default Roles and Capabilities.

If you'd like to make this change on your own site, you can do it in a child theme, create a custom plugin, or use a third party roles modification plugin.
